service/incremental-jsonsystem.py

```diff
@@ -1,4 +1,4 @@
-from flask import Flask, request, abort
+from flask import Flask, request, abort,send_from_directory
```

Comment:
need a space after the comma in "abort,send_from_directory"

Reply:
fixed